extensions/arc/src/models/postgresModel.ts

```diff
@@ -93,8 +94,8 @@ export class PostgresModel extends ResourceModel {
 			configuration.push(`${ramLimit ?? ramRequest!} ${loc.ram}`);
 		}
 
-		if (storage) {
-			configuration.push(`${storage} ${loc.storagePerNode}`);
+		if (dataStorage || logStorage) {
+			configuration.push(`${dataStorage ?? 0} data ${logStorage ?? 0} log ${loc.storagePerNode}`);
```

Comment:
I don't think it should ever show 0.  The data and log volumes always have some size.  But `storage.data.size` and `storage.logs.size` are optional in the spec.  The backend will choose some defaults if they aren't specified.  Eventually we'll use mutating webhooks to stick those defaults in the spec.  For now I would only show the storage sizes that are defined in the spec.  I'd add the backup volumes too (though those will go away when the backup service is implemented).

Reply:
If omitted, it defaults to the storage sizes used by the data controller.  But I don't love the idea of the client needing to know that.